packages/react/src/components/header/Header.stories.tsx

```diff
@@ -203,3 +239,41 @@ export const WithNavigationMenu = (args) => (
     </Header>
   </>
 );
+
+export const WithActionBar = (args) => {
+  return (
+    <Header {...args} onDidChangeLanguage={languageChangedAction}>
+      <StoryWIPAlert />
+
+      <Header.ActionBar
+        title="Helsingin kaupunki"
+        titleAriaLabel="Helsingin kaupunki"
+        titleUrl="https://hel.fi"
+        titleStyle={TitleStyleType.black}
+        onMenuButtonClick={(event) => {
+          // TODO
+          event.stopPropagation();
+        }}
+      >
+        <Header.NavigationLanguageSelector languages={languages}>
```

Comment:
Language selector should disappear on breakpoint S and go to next line ([source](https://app.abstract.com/projects/bf7025a0-5b50-11e9-80c2-5323106dd058/branches/958cf4cb-b29b-4eab-aa1c-0a7a892ce6e1/commits/latest/files/759b250d-52d4-4728-ab73-cfe0436f060c/layers/BABDD57B-2863-4F02-9160-26FAA90351EA?collectionId=e07fc52a-9dee-4617-946f-5dd1ee787196&collectionLayerId=06da2ff1-f946-45b0-8764-ea899d6ebf1d&mode=build&selected=1853857886-3AC769EA-F08D-4B6E-962C-8191A55D8C4E))